Merge changes from master

ralphsomeday opened the pull request:
<!--
Hello 👋 Thank you for submitting a pull request.

To help us merge your PR, make sure to follow the instructions below:

- Create or update the documentation. (Should be made against the documentation branch)
- Create or update the tests.
- Refer to the issue you are closing in the PR description - fix #issue
- Specify if the PR is in WIP (work in progress) state or ready to be merged

Please ensure you read through the Contributing Guide: https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md
-->

### What does it do?

Describe the technical changes you did.

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

ralphsomeday commented:
Thanks @JiboStore !

JiboStore commented:
Thanks to you indeed! Otherwise I'd still be lost with the redirect URL, not sure where to look for stuff as the documentation is totally unreliable (not updated)

Now that the redirect is solved, I still can't login to strapi successfully because the next step, Provider.js need to call getProfile() and it is using Simov's Purest library. The library has numerous predefined config from tons of providers except cognito!!

Did you by any chance have the entire login flow solved?

ralphsomeday commented:
So what happens after you authenticate with Cognito you should be redirected to your web app (front end) and you should get an access token, id token and a refresh token (see screenshot) as query parameters in your callback url. With the Bearer id token you should be able to get the user profile. Does this help?

![image](https://user-images.githubusercontent.com/61593456/92581139-8db64800-f28f-11ea-8970-4a4cde791287.png)

petersg83 committed:
add previousDefinition in migration options

petersg83 committed:
add migration for SQL DBs

petersg83 committed:
add migration for mongo DB

petersg83 committed:
refacto

petersg83 committed:
refacto

petersg83 committed:
refacto

petersg83 committed:
split code into files

petersg83 committed:
WIP

petersg83 committed:
refacto

soupette committed:
Merge pull request #9763 from strapi/i18n/rbac-locale-in-array

[i18n] add the locale column in the CM list view

soupette committed:
Fix middleware

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix single types

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix ST delete

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9762 from strapi/i18n/cm-common-fields

[I18n] Add middleware to get common fields

soupette committed:
Merge branch 'i18n/rbac-rework-front' of github.com:strapi/strapi into i18n/fix-single-types

soupette committed:
Merge pull request #9765 from strapi/i18n/fix-single-types

[I18n] Fix single types

petersg83 committed:
refacto

alexandrebodin committed:
Update ST localizations through core api update

petersg83 committed:
fix rebase

Commit:
[i18n] fix padding in locale picker list

soupette committed:
Merge pull request #9777 from strapi/features/i18n-fix-locale-padding

[i18n] fix padding in locale picker list

Commit:
[i18n/rbac] show locale picker even when not authorized

soupette committed:
Fix tests

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9778 from strapi/i18n/rbac-i18n-show-dropdown-when-locale-is-not-authorized

[i18n/rbac] show locale picker even when not authorized

Commit:
[i18n] don't add locale in settings when it already exists (#9776)

alexandrebodin committed:
[i18n] Improve localization  file sanitizer (#9766)

* Improve file sanitizer in create localizations

* Fix typo

petersg83 committed:
add comment

petersg83 committed:
refacto

alexandrebodin committed:
Fix typos

alexandrebodin committed:
Merge pull request #9768 from strapi/i18/st-core-api

[I18n] Single type core api update

petersg83 committed:
handle delete ST core-api (#9781)

alexandrebodin committed:
Merge pull request #9497 from strapi/i18n/disable-i18n-field

I18n/disable i18n field

Commit:
[I18n/rbac] add additional infos in delete modal (#9795)

Commit:
[i18n] add filters from the start (#9796)

Commit:
Add utils to CM folder in helper plugin

soupette committed:
Merge pull request #9799 from strapi/i18n/rbac-rework-front-utils-cm

Add utils to CM folder in helper plugin

alexandrebodin committed:
Merge branch 'master' into features/i18n

soupette committed:
Fix menu links that were missing when updating a content type

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9812 from strapi/i18n/fix-ctb-menu-links

Fix menu links that were missing when updating a content type

soupette committed:
Merge branch 'features/i18n' of github.com:strapi/strapi into i18n/rbac-rework-front

soupette committed:
Merge pull request #9568 from strapi/i18n/rbac-rework-front

[i18n] Front-end

petersg83 committed:
fill non localized fields at CM locale creation

petersg83 committed:
add migration when a CT becomes localized

alexandrebodin committed:
Merge pull request #9816 from strapi/i18n/fill-empty-attributes-at-creation

[i18n] Fill non localized fields at CM locale creation

petersg83 committed:
refacto

soupette committed:
Fix react select option style

Signed-off-by: soupette <cyril.lpz@gmail.com>

alexandrebodin committed:
Merge pull request #9825 from strapi/i18n/migration-default-locale

Add migration when a CT becomes localized

soupette committed:
Move links search logic into the main menu

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Add missing tests

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Add logic to update the menu links when the configurations have been changed

Signed-off-by: soupette <cyril.lpz@gmail.com>

Convly committed:
Sync super admin permissions with locales list for localized subjects

soupette committed:
Fix common fields

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9834 from strapi/i18n/front-fixes

I18n/front fixes

Convly committed:
I18n/ permissions rework (#9535)

* Add a domain layer for the permission, rework the engine handling of the permissions

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

* Add permissions-fields-to-properties migration for the admin

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

* Removes useless console.log

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

* Remove debug logLevel from provider-login.test.e2e.js

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

* Adds the new layout for the GET permissions, allow to subscribe to actionRegistered events, adds i18n handlers

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

* Fix typo

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

* Update permissions validators

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

* Update unit tests

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

* Update integrations test + fix some validation issues

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

* Change plugins & settings section format for the permissions layout

* only return locales property to localized subjects for the permission's layout

* Do not send the locales property to the permission's layout when there is no locales created

* Add the 'locales' property to publish & delete routes

* Fix unwanted mutation of the sections builder states on multiple builds

* Fix units tests with (new engine)

* Fix admin-role e2e test - Add locales property to the update payload

* fix e2e testsé

* Update e2e snapshots

* Fix unit test for i18n bootstrap

* Add mocks for i18n/bootstrap test

* Fix has-locale condition & updatePermission validator

* Avoid mutation in migration, always authorize super admin for has-locales condition

* Rework rbac domain objects, add a hook module and a provider factory

* Remove old providers

* Update the admin services & tests for the new rbac domain & providers

* Fix tests, bootstrap functions & services following rbac domain rework

* Update migration runner

* PR comments

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

* Remove useless console.log

* Fix sanitizeCondition bug

* Section builder rework

* Add test for the section-builder section & add jsdoc for the permission domain

* pr comments (without the migrations)

* fix fields-to-properties migration

* Add jsdoc for the sections-builder

* Moves createBoundAbstractDomain from permission domain to the engine service

* Remove debug logLevel for admin role test (e2e)

* Fix core-store

* Fix hooks & move business logic from i18n bootstrap to dedicated services

* add route get-non-localized-fields

* use write and read permission

* refacto

* add input validator

* add route doc

* handle ST

Co-authored-by: Pierre Noël <petersg83@gmail.com>
Co-authored-by: Alexandre BODIN <alexandrebodin@users.noreply.github.com>

alexandrebodin committed:
Merge branch 'master' into features/i18n

petersg83 committed:
add migration when i18n is disabled on a CT

Convly committed:
Set default jest timeout to 60s for every suite

Convly committed:
Fix permissions differences on resetSuperAdminPermissions

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

Convly committed:
Fix typo

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

Convly committed:
Fix resetSuperAdminPermissions unit test

Convly committed:
Use a global timeout instead of redefining it everytime

petersg83 committed:
use dropTableIfExists

Convly committed:
Chore/tests cleanup (#9857)

* Use new jest runner

* Set loglevel to fatal

* Update snapshots

* Remove update option from cli

Co-authored-by: Alexandre Bodin <bodin.alex@gmail.com>

Convly committed:
Fix ee e2e test for admin-roles

soupette committed:
Add more types to the localisation

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix Roles alignment

Signed-off-by: soupette <cyril.lpz@gmail.com>

alexandrebodin committed:
Merge pull request #9835 from strapi/i18n/super-admin-default-permissions

[i18N] Sync super admin permissions with locales list for localized subjects

alexandrebodin committed:
Merge pull request #9843 from strapi/i18n/migration-disable-CT

Add migration when i18n is disabled on a CT

petersg83 committed:
add test _locale filter on /relations/ route

soupette committed:
Fix redirection issue when deleting an entry

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix tabs border radius

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Use similar dropdown indicator as in the CM

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix create slug bug

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Change tooltip in helper component

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix redirection link

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Upgrade Buffet.js

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix version number hover css

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Change settings confirm modal type & change copy locale submit button label

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Update plugin name

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix Pr feedback

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9859 from strapi/i18n/fix-ctb

[i18n] Fix localisable attributes

soupette committed:
Merge pull request #9862 from strapi/i18n/cm-fix-redirection-after-delete

[I18n] CM fix redirection after delete

soupette committed:
Fix dropdown indicator

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix PR feedback

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9863 from strapi/i18n/fix-rbac-ui

Fix RBAC UI

soupette committed:
Merge pull request #9867 from strapi/i18n/fix-label-tooltip

[I18n] Fix label tooltip

soupette committed:
Merge pull request #9868 from strapi/ui/change-settings-modal

UI/change settings modal

Commit:
Fix loading information on error when adding a locale that already exists (#9875)

Commit:
Show all locales in the listview content-available cell (#9877)

Convly committed:
Remove permissions with invalid properties on startup

Signed-off-by: Convly <jean-sebastien.herbaux@epitech.eu>

petersg83 committed:
fix missing published_at in getNonLocalizedFields Route

Convly committed:
Display error message on the frontend (on permission save validation error)

soupette committed:
Change plugin i18n logo

Signed-off-by: soupette <cyril.lpz@gmail.com>

alexandrebodin committed:
Merge pull request #9879 from strapi/i18n/add-published_at-on-getNonLocalizedFields-route

fix missing published_at in getNonLocalizedFields route

Convly committed:
Move findPermissionsIdToRemove logic to a dedicated function

Commit:
[I18N] fix typo

Commit:
Fix margin in table checkbox (#9883)

Commit:
[I18N] fix padding in locale dropdown (#9884)

Commit:
Add border-radius to the list of locales in settings (#9885)

Commit:
Add spacing at the bottom of the user profile page (#9886)

Commit:
[I18N] fix typo (#9887)

Commit:
Display locale code instead of name when name is not defined (#9888)

Commit:
Fix bullet position in delete content for a locale modal (#9889)

Commit:
Fix margin for cog in ListView dropdown (#9890)

soupette committed:
Merge pull request #9891 from strapi/fix-typo

[I18N] fix typo

soupette committed:
Add search params to publish action

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9893 from strapi/i18n/add-qs-to-cm-actions

Add search params to publish action

soupette committed:
Add padding

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix media label css

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9894 from strapi/rbac/fix-padding

Add padding

soupette committed:
Fix border radius in RBAC Editview

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9895 from strapi/rbac/fix-boder-radius

Fix border radius in RBAC Editview

alexandrebodin committed:
Merge pull request #9881 from strapi/i18n/remove-breaking-permissions

Remove breaking permissions

Commit:
Fix margin on cog (again) (#9905)

Commit:
Fix centering issue on delete locale modale (#9904)

soupette committed:
Change Profile page dom

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Regenerate menu after changing the default locale

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Update permissions after creating a modal

Signed-off-by: soupette <cyril.lpz@gmail.com>

Commit:
Fix empty locale picker on listview (#9906)

soupette committed:
Merge pull request #9908 from strapi/i18n/fix-profile-page

Change Profile page dom

soupette committed:
Fix tests

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix PR feedback

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9909 from strapi/i18n/fix-locales-settings-update

I18n/fix locales settings update

soupette committed:
Fix hover state strapi link

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Only displayed allowed with rbac locales

Signed-off-by: soupette <cyril.lpz@gmail.com>

Commit:
Bumping buffet and fixing list radius (#9911)

soupette committed:
Fix Pr feedback

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix localizations display

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Only display not used locales

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Rename hook

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9914 from strapi/i18n/remove-already-added-locales

Only display not used locales

soupette committed:
Merge pull request #9912 from strapi/i18n/display-locale-rbac

Only display allowed locales with rbac

soupette committed:
Fix ctb menu upate

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix css

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9918 from strapi/fix/menu-links

i18n small fixes

soupette committed:
Fix role update when no conditions and redirect the user to hp in listview when response is 403

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix tooltip i18n

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9922 from strapi/cm/lv-fixtooltip

Fix tooltip i18n

soupette committed:
Merge pull request #9921 from strapi/cm/fix-lv-redirections

Fixes

petersg83 committed:
Delete all entries in a locale when locale is deleted

Commit:
Fix dropdown size for the user (#9931)

Commit:
Mock buffet core

soupette committed:
Merge pull request #9933 from strapi/fix-tests-locale-cells

Fix locale cells tests

soupette committed:
Fix locale picker

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge branch 'features/i18n' of github.com:strapi/strapi into features/i18n

petersg83 committed:
handle localized media attr in CM et CTB

petersg83 committed:
refacto

petersg83 committed:
fix test

alexandrebodin committed:
Merge pull request #9932 from strapi/i18n/migration-delete-locale

Delete all entries in a locale when locale is deleted

soupette committed:
Remove create permissions for i18n settings

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9940 from strapi/i18n/fix-settings

Remove create permissions for i18n settings

petersg83 committed:
refacto

petersg83 committed:
fix field migration not supposed to run

soupette committed:
Improve CM error management

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Ensure the LV always does not have an empty query

Signed-off-by: soupette <cyril.lpz@gmail.com>

alexandrebodin committed:
Update content-types.js

petersg83 committed:
fix unit test

alexandrebodin committed:
Merge pull request #9939 from strapi/i18n/better-handle-media-attr

Handle localized media attr in CM et CTB

alexandrebodin committed:
Merge branch 'master' into features/i18n

soupette committed:
Fix PR feedback

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #9942 from strapi/chore/fix-error-plugins

Improve CM error management

soupette committed:
Upgrade react-redux

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Ensure unique fields are localized

Signed-off-by: soupette <cyril.lpz@gmail.com>

alexandrebodin committed:
UID is always localized in i18n

alexandrebodin committed:
Merge pull request #9971 from strapi/i18n/remove-uid-field-localization

Ensure unique fields are localized

soupette committed:
Merge pull request #9966 from strapi/chore/upgrade-redux

Upgrade react-redux

alexandrebodin committed:
Update content-types.js

alexandrebodin committed:
init

alexandrebodin committed:
Merge pull request #9972 from strapi/i18n/uid-localized

UID should be localized in i18n

soupette committed:
Uid fields are localised in CM

Signed-off-by: soupette <cyril.lpz@gmail.com>

petersg83 committed:
rename isLocalized to isLocalizedContentType

alexandrebodin committed:
Use objects instead of strings to declare queries & mutattions

alexandrebodin committed:
Merge pull request #9985 from strapi/i18n/show-localised-uid

Uid fields are localised in CM

alexandrebodin committed:
Merge branch 'master' into features/i18n

alexandrebodin committed:
Support extra _ root params

Commit:
Ellipsis on locale cell only

alexandrebodin committed:
Add tests & handle _or & _where

soupette committed:
Merge pull request #9995 from strapi/fix-overflow-image

Ellipsis on locale cell instead of all cells in ListView

petersg83 committed:
handle compo, dz and media migration when disabling i18n on a field

alexandrebodin committed:
use globalId for the typeName

alexandrebodin committed:
Return sections based on role

alexandrebodin committed:
all return all actions in ee

soupette committed:
Fix sticky header and select the first locale for copy

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Fix modal confirm border radius

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
Merge pull request #10006 from strapi/fix/sticky-header

Fix sticky header and select the first locale for copy

soupette committed:
Fix jumpy app when no edition of a locale

Signed-off-by: soupette <cyril.lpz@gmail.com>

alexandrebodin committed:
Remove unnecessary code

petersg83 committed:
refacto

petersg83 committed:
refacto

alexandrebodin committed:
Merge pull request #9986 from strapi/i18n/naming-refacto-isLocalized

rename isLocalized to isLocalizedModel

alexandrebodin committed:
Merge pull request #9997 from strapi/i18n/disable-localized-media-compo-dz

Handle compo, dz and media migration when disabling i18n on a field

alexandrebodin committed:
Merge pull request #9994 from strapi/i18n/graphql

I18n/graphql

alexandrebodin committed:
Update naming & simplify filter

alexandrebodin committed:
Merge pull request #9999 from strapi/i18n/rbac-author-ce

I18n/rbac author ce

alexandrebodin committed:
Merge branch 'master' into features/i18n

petersg83 committed:
Fix bulkDelete on entites that are not in default locale

petersg83 committed:
refacto

alexandrebodin committed:
Merge pull request #10008 from strapi/i18n/fix-bulkDelete

Fix bulkDelete on entites that are not in default locale

petersg83 committed:
use strapi service

alexandrebodin committed:
Merge pull request #9861 from strapi/i18n/filter-relation-route

add test _locale filter on /relations/ route

petersg83 committed:
fix singletype put

petersg83 committed:
fix getNonLocalizedAttributes route for ST

alexandrebodin committed:
Merge pull request #10021 from strapi/i18n/fix-getNonLocalizedAttributes-st

fix getNonLocalizedAttributes route for ST

alexandrebodin committed:
Pass extra args in ctx.query for every mutation

alexandrebodin committed:
Target shadowcrud mutations only

alexandrebodin committed:
Merge pull request #10041 from strapi/fix/single-type-delete

[i18n] Fix single type delete

soupette committed:
Upgrade buffet.js to 3.3.5

Signed-off-by: soupette <cyril.lpz@gmail.com>

soupette committed:
update snapshots

Signed-off-by: soupette <cyril.lpz@gmail.com>

alexandrebodin committed:
Make i18n preinstalled (#10071)

soupette committed:
Enable i18n on media field when i18n is activated

Signed-off-by: soupette <cyril.lpz@gmail.com>

alexandrebodin committed:
Merge pull request #10074 from strapi/i18n/fix-media

I18n/fix media

alexandrebodin committed:
Merge branch 'master' into features/i18n

alexandrebodin committed:
Add an env var to set the init locale code on startup

alexandrebodin committed:
Fix typo

alexandrebodin committed:
Merge pull request #10081 from strapi/i18n/allow-env-var-default-locale

Add an env var to set the init locale code on startup

alexandrebodin committed:
Merge branch 'master' into features/i18n

alexandrebodin committed:
Merge branch 'master' into features/i18n

alexandrebodin committed:
Fix graphql naming to use the same input namine convention as the plugin

alexandrebodin committed:
Merge pull request #10086 from strapi/i18n/fix-plural-name

Fix graphql naming to use the same input naming convention as the graphql plugin

alexandrebodin committed:
Merge branch 'master' into features/i18n

alexandrebodin committed:
Add i18n telemetry

alexandrebodin committed:
Add clean stringify

dependabot[bot] committed:
Bump is-docker from 2.2.0 to 2.2.1 (#10087)

Bumps [is-docker](https://github.com/sindresorhus/is-docker) from 2.2.0 to 2.2.1.
- [Release notes](https://github.com/sindresorhus/is-docker/releases)
- [Commits](https://github.com/sindresorhus/is-docker/compare/v2.2.0...v2.2.1)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

alexandrebodin committed:
Merge pull request #10092 from strapi/i18n/telemetry

Add i18n telemetry

alexandrebodin committed:
Merge branch 'master' into features/i18n

alexandrebodin committed:
Merge pull request #10000 from strapi/features/i18n

Features/i18n

alexandrebodin committed:
v3.6.0

dependabot[bot] committed:
Bump codemirror from 5.60.0 to 5.61.0

Bumps [codemirror](https://github.com/codemirror/CodeMirror) from 5.60.0 to 5.61.0.
- [Release notes](https://github.com/codemirror/CodeMirror/releases)
- [Changelog](https://github.com/codemirror/CodeMirror/blob/master/CHANGELOG.md)
- [Commits](https://github.com/codemirror/CodeMirror/compare/5.60.0...5.61.0)

Signed-off-by: dependabot[bot] <support@github.com>

soupette committed:
Merge pull request #10099 from strapi/dependabot/npm_and_yarn/codemirror-5.61.0

Bump codemirror from 5.60.0 to 5.61.0

dependabot[bot] committed:
Bump @sentry/node from 6.2.5 to 6.3.0 (#10100)

Bumps [@sentry/node](https://github.com/getsentry/sentry-javascript) from 6.2.5 to 6.3.0.
- [Release notes](https://github.com/getsentry/sentry-javascript/releases)
- [Changelog](https://github.com/getsentry/sentry-javascript/blob/master/CHANGELOG.md)
- [Commits](https://github.com/getsentry/sentry-javascript/compare/6.2.5...6.3.0)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump aws-sdk from 2.888.0 to 2.890.0 (#10101)

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.888.0 to 2.890.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.888.0...v2.890.0)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump aws-sdk from 2.890.0 to 2.892.0 (#10136)

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.890.0 to 2.892.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.890.0...v2.892.0)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump pg from 8.5.1 to 8.6.0 (#10113)

Bumps [pg](https://github.com/brianc/node-postgres/tree/HEAD/packages/pg) from 8.5.1 to 8.6.0.
- [Release notes](https://github.com/brianc/node-postgres/releases)
- [Changelog](https://github.com/brianc/node-postgres/blob/master/CHANGELOG.md)
- [Commits](https://github.com/brianc/node-postgres/commits/pg@8.6.0/packages/pg)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump chalk from 4.1.0 to 4.1.1 (#10112)

Bumps [chalk](https://github.com/chalk/chalk) from 4.1.0 to 4.1.1.
- [Release notes](https://github.com/chalk/chalk/releases)
- [Commits](https://github.com/chalk/chalk/compare/v4.1.0...v4.1.1)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump @babel/runtime from 7.13.10 to 7.13.17 (#10091)

Bumps [@babel/runtime](https://github.com/babel/babel/tree/HEAD/packages/babel-runtime) from 7.13.10 to 7.13.17.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.13.17/packages/babel-runtime)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump @babel/core from 7.13.15 to 7.13.16 (#10097)

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.13.15 to 7.13.16.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.13.16/packages/babel-core)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

petersg83 committed:
[graphQL + mongo] Fix broken localizations query (#10141)

* fix localizations resolver

* Revert "fix localizations resolver"

This reverts commit 31c92e6380036d29b2008324586ed8e5f082e4f5.

* fix localizations resolver by adding _id to populate option

gfpacheco committed:
Fix #10125 - Accept falsy values returned by controllers in GraphQL queries and mutations (#10145)

* Fix #10125 - accept falsy values returned by controllers

* Add test for falsy ctx.body on built graphql resolver

* Use test instead of it in unit tests

alexandrebodin committed:
Fix issue relation loosing extra options on ctb update Fixes #3409 (#10102)

tomdyson committed:
Fix typo in query error message (#10116)

The possessive `its` doesn't have an apostrophe.

https://www.dictionary.com/e/its-vs-its/

cogor committed:
Fixed typo (#10132)

cogor committed:
Updated Russian translate (#10133)

SorinGFS committed:
Add DKIM config example (#9923)

Convly committed:
[RBAC] Ignore only non visible attributes instead of non configurable ones (#10139)

* Ignore only non visible attributes instead of non configurable ones

* Update snapshots

* Refacto buildDeepAttributesCollection

TheSeally committed:
Fix issue with incorrect cli termination when using flags quickstart and no-run (#10111)

On Windows, an empty return from the createQuickStartProject function caused the process termination to hang.
Added explicit process termination for this platform

alexandrebodin committed:
Remove cache key so the url is valid for cors policy (#10148)

dependabot[bot] committed:
Bump apollo-server-koa from 2.22.1 to 2.23.0 (#10155)

Bumps [apollo-server-koa](https://github.com/apollographql/apollo-server/tree/HEAD/packages/apollo-server-koa) from 2.22.1 to 2.23.0.
- [Release notes](https://github.com/apollographql/apollo-server/releases)
- [Changelog](https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md)
- [Commits](https://github.com/apollographql/apollo-server/commits/apollo-server-koa@2.23.0/packages/apollo-server-koa)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump snyk from 1.551.0 to 1.566.0 (#10153)

Bumps [snyk](https://github.com/snyk/snyk) from 1.551.0 to 1.566.0.
- [Release notes](https://github.com/snyk/snyk/releases)
- [Changelog](https://github.com/snyk/snyk/blob/master/.releaserc)
- [Commits](https://github.com/snyk/snyk/compare/v1.551.0...v1.566.0)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump ioredis from 4.26.0 to 4.27.1 (#10165)

Bumps [ioredis](https://github.com/luin/ioredis) from 4.26.0 to 4.27.1.
- [Release notes](https://github.com/luin/ioredis/releases)
- [Changelog](https://github.com/luin/ioredis/blob/master/Changelog.md)
- [Commits](https://github.com/luin/ioredis/compare/v4.26.0...v4.27.1)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump @testing-library/jest-dom from 5.11.10 to 5.12.0 (#10166)

Bumps [@testing-library/jest-dom](https://github.com/testing-library/jest-dom) from 5.11.10 to 5.12.0.
- [Release notes](https://github.com/testing-library/jest-dom/releases)
- [Changelog](https://github.com/testing-library/jest-dom/blob/main/CHANGELOG.md)
- [Commits](https://github.com/testing-library/jest-dom/compare/v5.11.10...v5.12.0)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
